pycqed/analysis_v2/timedomain_analysis.py

```diff
@@ -1640,7 +1676,8 @@ def prepare_projected_data_plot(
             sweep_points = self.proc_data_dict['sweep_points_dict'][qb_name][
                 'sweep_points']
         plot_names_cal = []
-        if plot_cal_points and self.num_cal_points != 0:
+        if plot_cal_points and self.num_cal_points != 0 and \
+                len(self.cal_states_dict):
```

Comment:
What is the difference between num_cal_points and the len of the cal states dict?

Reply:
I answered this in the context of another question but here it is again.

num_cal_points counts how many calibration segments were used in the experiment. So it will be nonzero even if no calibration point information was provided (reflected by self.cal_states_dict = {qbn: {} for qbn in self.qb_names}).

Reply:
Added now